### ISSUE TITLE
trian one class with part of tiny-yolov2 

### ISSUE THREAD

**Issue by bonne658** [AlexeyAB/darknet#5276]
I run model on a device  that has 8M memory. So I delete two convolutional layer that have 1024 kernels and the pooling layer between them. Besides, I use 11 layers pre-train weights and trian with my own dataset which has **one class**. Before 1000 iterations, loss < 0.06, mAP > 99%, but the best model detects many other objects as the trained class. Is there any suggestions?
Thank you for apply!
my cfg is:(change width, height and anchor)
`[net]
batch=64
subdivisions=8
width=320
height=224
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
max_batches = 40200
policy=steps
steps=-1,100,20000,30000
scales=.1,10,.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
size=1
stride=1
pad=1
filters=30
activation=linear

[region]
anchors = 1.93,2.25,  3.18,3.59,  3.56,5.28,  4.90,6.53,  6.40,5.78
bias_match=1
classes=1
coords=4
num=5
softmax=1
jitter=.2
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .6
random=1`
